docs/reference/api.md

```diff
@@ -55,8 +55,9 @@ The Ops library for writing and testing Juju charms includes a [Python client fo
 
 ```python
 import ops
-client = ops.pebble.Client('/path/to/.pebble.socket')
-client.start_services(['svc1', 'svc2'])
+client = ops.pebble.Client("/path/to/.pebble.socket")
+changeID = client.stop_services(["mysvc"])
+client.wait_change(changeID)
```

Comment:
`wait_change` isn't needed here, as the Python client's `stop_services` (and similar) does it automatically. See [here](https://github.com/canonical/operator/blob/cece866bada2d9ada7a240846123a1c67faf48ee/ops/pebble.py#L2242). Let's remove it but mention that explicitly in a comment:
```suggestion
client.stop_services(["mysvc"])  # Python client also waits for change to finish
```